peps/pep-0780.rst

```diff
@@ -0,0 +1,469 @@
+build and runtime dependencies to be compatible. Capturing the first version of
+a dependency that is compatible precisely in metadata is currently not
+possible, and increasing the minimum version of a dependency also for the
+GIL-enabled build is usually undesirable since it unnecessarily limits
+compatibility between packages.
+
+Some concrete examples of such issues have been discussed in the `Environment
+marker for free-threading`_ Discourse thread:
+
+- Cython has (experimental) support for free-threading only in its master
+  branch, and is used by a lot of projects that already publish ``cp313t``
+  wheels. Picking up the wrong Cython version is causing a lot of obscure build
+  failures and runtime crashes. It would be quite beneficial if the metadata
+  could express that (c.f. `Require Cython Pre-release for Free-Threaded
+  Python`_).
+- CFFI has no support for free-threading yet, and `Armin Rigo, one of the
+  maintainers, has stated
+  <https://github.com/python-cffi/cffi/pull/143#issuecomment-2580781899>`__
+  that it may be a good idea to fork ``cffi``, implement support for
+  free-threading, and only come back to the CFFI project with a single large PR
+  that adds support after the functionality "is reasonably well-tested (either
+  as tests or, better in this case, tested by being in use in various other
+  projects)". There are a lot of projects that depend on ``cffi``. They are
+  likely fine to start depending on a fork for free-threading only, however
+  depending on a fork for >=3.13 or for all Python versions seems like a much
+  larger ask, and more disruptive for distribution packagers.
+
+While these concrete examples may be addressed later this year by Cython and
+CFFI making compatible releases, the same issue is going to repeat further up
+the stack. The free-threading rollout is expected to take several years, and an
+environment marker for free-threading will make that rollout significantly less
+bumpy.
+
+Another important ABI feature that is not yet covered by environment markers is
+the bitness of the interpreter. In most cases, the ``sys_platform`` or
+``platform_system`` markers are enough, because there's only a single bitness
+in use per platform. This is not the case on Windows however: both 32-bit and
+64-bit Python interpreters are widely used on x86-64 Windows. Not being able to
+distinguish between the two may be relevant for packages that provide C
+extensions. For example, SciPy does not provide ``win32`` wheels (it isn't able
+to due to the lack of a suitable 32-bit compiler toolchain with Fortran
+support). Those wheels lacking can be awkward especially for projects where
+SciPy is an optional dependency only. In that case, it would be useful to be
+able to specify that SciPy is required *unless* the interpreter is a 32-bit one
+on Windows (c.f. `Require SciPy Unless on 32-bit win32`_) - it avoids users who
+install such packages to get failed from-source installs because of the missing
+wheels.
```

Comment:
The second half of the sentence (after '-') doesn't make sense.
```suggestion
SciPy is an optional dependency only. In that case, it would be useful to be
able to specify that SciPy is required *unless* the interpreter is a 32-bit one
on Windows (c.f. `Require SciPy Unless on 32-bit win32`_), to avoid failed
from-source installations due to the missing wheels.
```